spec/features/journeys/happy_paths/dfe_sign_in_spec.rb

```diff
@@ -0,0 +1,109 @@
+require "rails_helper"
+
+# To update the id_token fixtures using real responses:
+# you will need to have 2 user accounts, onw with a TRN and one without
+# 1. remove the old fixtures from spec/fixtures/omniauth_hashes/
+# 2. in .env.test:
+#   comment out HOSTING_DOMAIN
+#   comment out the existing TRA_OIDC_* env vars, and copy the 5 TRA_OIDC_* env vars from .env
+# 3. set `regenerate_fixtures` to true below
+# 4. update `email_for_account_with_trn` to the email address for the account with a TRN
+# 5. update `email_for_account_without_trn` to the email address for the account without a TRN
+# 6. make sure your local server is not running on port 3000
+# 7. run the first 2 specs - be sure to follow the prompts from the command-line, and do not enter your code into the browser
+RSpec.feature "DfE Sign In", :with_default_schedules, type: :feature do
+  include Helpers::JourneyAssertionHelper
+  include Helpers::JourneyStepHelper
+
+  before do
+    WebMock.disable_net_connect!(allow_localhost: true, allow: URI(ENV["TRA_OIDC_DOMAIN"]).host)
```

Comment:
This is only needed when regenerating fixtures, and needs to be 'put back' after the spec runs to avoid mutating the value for the rest of the test suite

Does it work if you use mocking of this somehow, or failing that using an `after` block